conda-forge.yml

```diff
@@ -7,7 +7,7 @@ build_platform:
 conda_forge_output_validation: true
 test: native_and_emulated
 # build_with_mambabuild: false
-upload_on_branch: main
+upload_on_branch: 1.15.0rc1
```

Comment:
For this to work, the target branch for merging needs to be `1.15.0rc1`. You'll need to create it manually first:

```sh
git checkout main
git checkout -b 1.15.0rc1
# assuming you name it origin
git push origin 1.15.0rc1
```